.gitlab-ci.yml

```diff
@@ -10,10 +10,28 @@ include:
 stages:
   - test
 
+test:check-commits:
+  except:
+    refs:
+      - /^(master|[0-9]+\.[0-9]+\.x)$/
+    variables:
+      - $RUN_TESTS_STAGING == "true"
+
+test:check-python3-formatting:
+  except:
+    refs:
+      - /^(master|[0-9]+\.[0-9]+\.x)$/
+    variables:
+      - $RUN_TESTS_STAGING == "true"
+
```

Comment:
Technically speaking, what we want is to skip these in `staging` branch (and for all repos, really) regardless of the magic variale. I will follow-up on a PR in mendertesting.

Reply:
Please have a look at this comment:
https://tracker.mender.io/browse/QA-186?focusedCommentId=109286&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-109286

I see value in running these tests from non-staging branches.